.github/actions/install-aiida-core/action.yml

```diff
@@ -0,0 +1,50 @@
+name: Install aiida-core
+description: Install aiida-core package and its Python dependencies
+
+inputs:
+  python-version:
+    description: Python version
+    default: '3.9'              # Lowest supported version
+    required: false
+  extras:
+    description: aiida-core extras (including brackets)
+    default: ''
+    required: false
+  # NOTE: Hard-learned lesson: we cannot use type=boolean here, apparently :-(
+  # https://stackoverflow.com/a/76294014
+  from-requirements:
+    description: Install aiida-core dependencies from pre-compiled requirements.txt file
+    default: 'true'
+    required: false
```

Comment:
Are these not mutually exclusive? If extras are specified, the install cannot be done from a requirements file? If so, we can just get rid of the `from-requirements` variable right and just check whether `extras` is not an empty string.

Reply:
Opened #6366